www/public-interface.js

```diff
@@ -223,8 +230,8 @@ module.exports = function init(exec, cookieHandler, urlUtil, helpers, globalConf
     return publicInterface.sendRequest(url, { method: 'options', params: params, headers: headers }, success, failure);
   };
 
-  function uploadFile(url, params, headers, filePath, name, success, failure) {
-    return publicInterface.sendRequest(url, { method: 'upload', params: params, headers: headers, filePath: filePath, name: name }, success, failure);
+  function uploadFile(url, params, headers, filePath, name, success, failure, progress) {
```

Comment:
Will this work for the https://github.com/danielsogl/awesome-cordova-plugins/ wrapper? I think it could be a problem but not sure, because usually the last two arguments are meant to be `success` and `fail` handlers. Would you please check?
If it causes problems we can keep the short-hand function `uploadFile` and make this functionality available only via `sendRequest`.